static/main.css

```diff
@@ -65,6 +65,20 @@ td:first-child {
   text-align: right;
 }
 
+/* Sticky header for the table. */
+th {
+    background: #333333;
+    box-shadow: 0px 2px 2px 0px rgb(0, 0, 0, 25%);
+    padding: 4px 2px;
+    position: -webkit-sticky;
```

Comment:
Isn't this `-webkit--position: sticky`?

Reply:
To be honest, it works in modern Chrome without the prefix, so I just kept the prefixed option I saw in an article on the subject for the heck of it.

Reply:
This is probably as official as it gets: https://developers.google.com/web/updates/2012/08/Stick-your-landings-position-sticky-lands-in-WebKit

We probably don't need it, want me to remove it?

Reply:
According to Can I Use, it's still useful for a non-negligible portion of iOS users: https://caniuse.com/css-sticky